src/components/calcite-block/calcite-block.scss

```diff
@@ -21,138 +28,147 @@
 }
 
 .header-container {
-  display: grid;
+  @apply grid items-stretch;
   grid-template: auto / auto 1fr auto;
   grid-template-areas: "handle header control";
   grid-column: header-start / control-end;
   grid-row: 1 / 2;
-  align-items: stretch;
 
   & > .header {
-    padding: var(--calcite-spacing-three-quarters) 0;
+    @apply py-3 px-0;
   }
 }
 
 .toggle {
-  display: flex;
-  align-items: center;
-  flex-wrap: nowrap;
-  margin: 0;
-  padding: var(--calcite-spacing-three-quarters) 0;
-  background-color: transparent;
-  border: none;
-  cursor: pointer;
-  font-family: inherit;
-  text-align: unset;
-  @include focus-style-base();
+  @apply
+    bg-transparent
+    flex
+    flex-no-wrap
+    font-inherit
+    items-center
+    m-0
+    py-3
+    px-0
+    border-none
+    cursor-pointer
+    text-left
```

Comment:
☝️ Sorry if this is a newb tailwind question, but does a `text-unset` class exist?

Reply:
I haven't seen one, but maybe if we need it a lot, it could be added to the config. Probs better to just add a standard CSS rule when it's needed.

Reply:
> Probs better to just add a standard CSS rule when it's needed.

:+1: edge cases should probably should not be added to the config. If an override is needed there may be an opportunity to refactor.